server/util/tracing/tracing.go

```diff
@@ -147,10 +150,20 @@ func Configure(configurator *config.Configurator, healthChecker interfaces.Healt
 		return bsp.Shutdown(ctx)
 	})
 
+	ctx, cancel := context.WithTimeout(context.Background(), resourceDetectionTimeout)
+	defer cancel()
+	res, err := resource.New(ctx,
+		resource.WithDetectors(&gcp.GKE{}, &gcp.GCE{}),
```

Comment:
can you:

gke := &gcp.GKE{}
resource.WithDetectors(gke, gke)

or do they need to be different?